src/multi_vector_simulator/A1_csv_to_json.py

```diff
@@ -312,7 +314,13 @@ def create_json_from_csv(
                     )
                 # add column specific parameters
                 if column == STORAGE_CAPACITY:
```

Comment:
You will also need to write/update the pytests for A1 to make sure your function works as intended.

Reply:
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/bbf88f91a26ec5eecd6a1755c3430f969a932320